app/controllers/user/users_controller.rb

```diff
@@ -1,7 +1,7 @@
-class User::UsersController < ApplicationController
+class User::UsersController < User::BaseController
```

Comment:
I like the change of inheritance to BaseController so we can require_user before each of these actions